src/utils/ssr/getAuthenticationInfo.js

```diff
@@ -97,14 +87,6 @@ const getAuthenticationInfo = async () => {
     { ...userPoolClientDetails, Domain: `${Domain}.auth.eu-west-1.amazoncognito.com` },
   );
 
-  /*
-    You can use this as a shim for authorization in the future:
-
-    const { Auth } = withSSRContext(context);
-    Auth.configure(amplifyConfig.Auth);
-    const user = await Auth.currentAuthenticatedUser();
-  */
-
   return {
```

Comment:
if we don't need this, delete the code

Reply:
it was removed, youre looking at a negative diff